CP_Source/Apps/Nutanix_Frame/README.md

```diff
@@ -149,63 +159,172 @@ CHECK_FOR_UPDATES_ON_STARTUP=OFF
 
-**Copy Client ID** and **Client secret** for later reference
+---
 
------
+## Frame SAT Kiosk Mode Profile
+```js
+import 'igel/frame-sat-kiosk-profile.xml'
+```
+
+The Frame SAT Kiosk Custom Profile is designed to support a specific end user workflow relying on Frame's SATs or [Secure Anonymous Tokens](https://docs.frame.nutanix.com/frame-apis/sat-api.html) for identity; this flow also assumes a particular Frame configuration to support KIOSK experience as defined below.
+
+### SAT Kiosk Mode User Experience:
+
+1. End users will not authenticate to a SAML2-based identity provider (this script uses the Frame Secure Anonymous Token (SAT) functionality for session authentication).
+2. User cache is removed prior to start of Frame App to ensure no user preference settings have persisted since the prior use of Frame App.
+3. Frame App will launch in "kiosk mode" (full screen).
+4. End users will be taken by Frame App directly to the desktop or application (depends on the [Launch Link](https://docs.frame.nutanix.com/session-conduct/advanced-integrations.html#launch-links) configuration).
+5. When a Frame session starts, the remote desktop will be in full-screen mode.
+6. When end users disconnect or close their session, Frame App will be restarted with a new SAT token. Disconnect behavior configurable with Frame Session Settings.
+
+### SAT + Kiosk configuration requirements:
+
+1. A *Published* Launchpad.
+2. API Provider configured at the Organization entity.
+3. Secure Anonymous Token Provider at the Account entity granting a role of Launchpad User for a specific Launchpad in a Frame account (under the Organization entity).
+4. Frame [Launch Link](https://docs.frame.nutanix.com/session-conduct/advanced-integrations.html#launch-links) is used, rather than a Launchpad URL to support automatic start of the user's session and to simplify the UX.
+5. *Optional:* The Frame account production workload VMs can be joined to a Windows domain, if desired.
+
+
+### Required Environment Variables
+
+The following environment variables must be configured in the IGEL Custom Profile for this profile to work.
+
+1. **Edit your IGEL UMS Custom Profile and go to**:
+   > System > Firmware Customization > Environment Variables > Predefined
+
+2. **Set the following environment variables:**
+   - `FRAME_CLIENT_ID` - obtained from the API provider when a set of API credentials are created.
+   - `FRAME_CLIENT_SECRET` - obtained from the API provider when a set of API credentials are created.
+   - 
+      `FRAME_SAT_URL` - URL obtainable from the Playground. 
+      
+      For example: `https://api.console.nutanix.com/v1/accounts/XXXXXXXX-XXXX-XXXX-XXXX-31d09e2881cd/secure-anonymous/secure-anon-XXXXXXXX-XXXX-XXXX-XXXX-c5e2dc93df1e/tokens`.
+   - `FRAME_ACCOUNT_ID` - obtainable from the FRAME_SAT_URL if the SAT Provider was created on the Frame Account.
```

Comment:
I've some better instructions for getting and account uuid.

Also... this would be a nice thing to show on an Account Summary page.